neurokit2/hrv/hrv_nonlinear.py

```diff
@@ -144,7 +144,62 @@ def hrv_nonlinear(peaks, sampling_rate=1000, show=False, **kwargs):
     Returns
     -------
     DataFrame
-        Contains non-linear HRV metrics.
+        DataFrame consisting of the computed non-linear HRV metrics, which includes:
+
+        .. codebookadd::
+            HRV_SD1|Standard deviation perpendicular to the line of identity. It is an index of \
+                short-term RR interval fluctuations, i.e., beat-to-beat variability. It is \
+                equivalent (although on another scale) to RMSSD, and therefore it is redundant to \
+                report correlation with both.
+            HRV_SD2|Standard deviation along the identity line. Index of long-term HRV changes.
+            HRV_SD1SD2|Ratio of SD1 to SD2. Describes the ratio of short term to long term \
+                variations in HRV.
+            HRV_S|Area of ellipse described by *SD1* and *SD2* (``pi * SD1 * SD2``). It is \
+                proportional to *SD1SD2*.
+            HRV_CSI|The Cardiac Sympathetic Index (Toichi, 1997) is a measure of cardiac \
```

Comment:
This would perhaps be a nice addition, however would probably make the docstring quite a bit longer. I think Heather did what you are talking about in her own manual codebook.

Reply:
Do you think it would be possible to add the references at the bottom of the docstring without increasing the length much? E.g. with a new directive "codebookrefadd"? This could also be a separate PR.

Reply:
It should be possible. I would create a different PR for this since I am not yet 100% sure how you would imagine this functionality so I would need more info on this